examples/getstarted/config/environments/development/security.json

```diff
@@ -1,30 +1,28 @@
 {
-  "csrf": {
-    "enabled": false,
-    "key": "_csrf",
-    "secret": "_csrfSecret"
-  },
   "csp": {
-    "enabled": false,
-    "policy": {
-      "default-src": "'self'"
-    }
+    "enabled": true,
+    "policy": [
+      {
+        "img-src": "'self' http:"
+      },
+      "block-all-mixed-content"
+    ]
   },
   "p3p": {
-    "enabled": false,
+    "enabled": true,
     "value": ""
```

Comment:
P3P on AdminUI states the value is required:

![image](https://user-images.githubusercontent.com/8593673/65320103-fb84d380-db55-11e9-948e-0567fe8326b2.png)

Reply:
OK I'll have to find a good default value then

Reply:
TBH P3P is a pretty outdated security thing to begin with :stuck_out_tongue_closed_eyes:

Reply:
I guess I can just disable it for now then

Reply:
:+1: